lib/java/desktop-environment/src/main/java/org/enso/desktopenvironment/TrashFactory.java

```diff
@@ -0,0 +1,28 @@
+package org.enso.desktopenvironment;
+
+final class TrashFactory {
```

Comment:
Too much classes for nothing? Why do we need to have `ThrashFactory`? Can the getter be in `Platform` or in `Thrash` itself?